module/proto/peggy/v1/attestation.proto

```diff
@@ -12,14 +10,24 @@ option go_package = "github.com/althea-net/peggy/module/x/peggy/types";
 // be handled
 enum ClaimType {
   option (gogoproto.goproto_enum_prefix) = false;
-
-  CLAIM_TYPE_UNKNOWN  = 0 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_UNKNOWN" ];
-  CLAIM_TYPE_DEPOSIT  = 1 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_DEPOSIT" ];
-  CLAIM_TYPE_WITHDRAW = 2 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_WITHDRAW"];
-  CLAIM_TYPE_ERC20_DEPLOYED = 3 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_ERC20_DEPLOYED"];
+  
+  CLAIM_TYPE_UNKNOWN             = 0 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_UNKNOWN" ];
+  CLAIM_TYPE_DEPOSIT             = 1 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_DEPOSIT" ];
+  CLAIM_TYPE_WITHDRAW            = 2 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_WITHDRAW"];
+  CLAIM_TYPE_ERC20_DEPLOYED      = 3 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_ERC20_DEPLOYED"];
   CLAIM_TYPE_LOGIC_CALL_EXECUTED = 4 [(gogoproto.enumvalue_customname) = "CLAIM_TYPE_LOGIC_CALL_EXECUTED"];
 }
 
+// ConfirmType is the cosmos type of a confirmation of a given object from the eth signers
+enum ConfirmType {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  CONFIRM_TYPE_UNKNOWN  = 1 [(gogoproto.enumvalue_customname) = "CONFIRM_TYPE_UNKNOWN"];
```

Comment:
repeat of the above.

Reply:
This should be 0 unless unknown here is an actual type that is used?